apps/web/src/components/FeatureBlock/FeatureBlock.tsx

```diff
@@ -18,6 +20,18 @@ interface FeatureBlockProps {
   };
 }
 
+interface ReactImageProps {
+  UrlArray: string[];
+}
+
+const ReactImage = ({ UrlArray }: ReactImageProps) => {
+  const { src } = useImage({
+    srcList: UrlArray,
+  });
+
+  return <img src={src} alt="" />;
+};
```

Comment:
This may be handy as a reusable component in `packages/ui` as something like:

This would also allow us to enforce an `alt` attribute.

```tsx

type ImgProps = Omit<React.ImgHTMLAttributes<HTMLImageElement>, "src" | "alt"> & {
  sources: string[];
  alt: string;
}

const Img = React.forwardRef<HTMLImageElement, ImgProps>(
  ({ sources, ...rest }, forwardedRef) => {
    const { src } = useImage({
      srcList: UrlArray,
    });
    
    return <img ref={forwardedRef} {...rest} src={src} />;
  },
);
```